src/styles/main.scss

```diff
@@ -0,0 +1,3 @@
+@import 'utilis/variables';
```

Comment:
Ensure that the file `utilis/variables.scss` exists and contains the definitions for all the variables used in `blocks/calendar.scss`, such as `$border-width`, `$padding`, `$column-width`, `$day-size`, and `$week-days`.